mysql-test/suite/rpl/r/rpl_bug-lp-1667307.result

```diff
@@ -0,0 +1,31 @@
+include/rpl_init.inc [topology=1->3,3->2,1->2]
```

Comment:
Since this is an upstream bug, use upstream bug number rpl.bug85371 for testcase naming (for easier upstream submission - see Confluence)